Dockerfile-alpine

```diff
@@ -0,0 +1,57 @@
+FROM php:7.0-alpine
+
+MAINTAINER Tobias Munk tobias@diemeisterei.de
```

Comment:
Is it okay to use `LABEL` now instead of `MAINTAINER`?

Because reading it [here](https://github.com/docker/docker/blob/master/docs/deprecated.md#maintainer-in-dockerfile), the deprecation start at v1.13.0, while v1.13.0 is currently in pre-release state (RC2).